### PR TITLE
Makes a gentle reminder on medigun usage

### PR DESCRIPTION
**Pull request by silicons**
Makes a gentle reminder on proper usage of mediguns for players who don't know yet.

### CODE REVIEW

**Comment by YakumoChen** [comment]
i ded

**Comment by TechnoAlchemisto** [comment]
first change is fine second change is hugbox and gross

**Comment by optimumtact** [comment]
:salt: :hugbox: :ided:

File: code/modules/projectiles/guns/medbeam.dm [hunk 1]
```diff
@@ -28,6 +28,9 @@
 	..()
 	LoseTarget()
 
+/obj/item/weapon/gun/medbeam/pickup(mob/living/user)
+	user << "<span class='userdanger'>Using this medical beamgun in close proximity with other medical beamguns is highly discouraged, as spontaneous explosions will occur if the beams ever come into contact.</span>"
```
Comment (optimumtact) [inline code/modules/projectiles/guns/medbeam.dm]: Gross as fuck, remove this immersion breaking shit

Comment (kevinz000) [inline code/modules/projectiles/guns/medbeam.dm]: How is this immersion breaking?

Comment (optimumtact) [inline code/modules/projectiles/guns/medbeam.dm]: Get it out

Comment (kevinz000) [inline code/modules/projectiles/guns/medbeam.dm]: Fine.

File: code/modules/projectiles/guns/medbeam.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/medbeam
 	name = "Medical Beamgun"
-	desc = "Delivers medical nanites in a focused beam."
+	desc = "Delivers medical nanites in a focused beam. On it is a label saying: 'Warning: Crossing the beam from this device with another is highly not recommended'."
```
Comment (Jalleo) [inline code/modules/projectiles/guns/medbeam.dm]: Wait what happens when they cross?

Comment (Bawhoppen) [inline code/modules/projectiles/guns/medbeam.dm]: >"what happens when they cross?"

Comment (Incoming5643) [inline code/modules/projectiles/guns/medbeam.dm]: >""happens when they cross?""

Comment (Super3222) [inline code/modules/projectiles/guns/medbeam.dm]: > """when they cross?"""

Comment (KorPhaeron) [inline code/modules/projectiles/guns/medbeam.dm]: This sentence is terribly clunky.

Comment (KorPhaeron) [inline code/modules/projectiles/guns/medbeam.dm]: Please write a better sentence mekhi

Comment (KorPhaeron) [inline code/modules/projectiles/guns/medbeam.dm]: Specifically "highly not recommended" is very bizzare.

Comment (optimumtact) [inline code/modules/projectiles/guns/medbeam.dm]: "Delivers medical nanites in a focused beam - crossing two or more beams is not recommended"

**Comment by kevinz000** [comment]
K hold on

**Comment by Iamgoofball** [comment]
Made a competing pr

**Comment by kevinz000** [comment]
@KorPhaeron  fixed @optimumtact

**Comment by Iamgoofball** [comment]
Merge my PR instead

**Comment by PKPenguin321** [comment]
ps @kevinz000 nice salt pr

**Comment by TechnoAlchemisto** [comment]
Fucjin owned

**Pull request closed without merging**